src/client/ui/select-element.js

```diff
@@ -94,7 +94,8 @@ function createOption (realOption, parent) {
     var isOptionDisabled = realOption.disabled || domUtils.getTagName(realOption.parentElement) === 'optgroup' &&
                                                   realOption.parentElement.disabled;
 
-    option.textContent = realOption.text;
+    // eslint-disable-next-line no-restricted-properties
+    nativeMethods.nodeTextContentSetter.call(option, realOption.text);
```

Comment:
Why we do not use native method to get `realOption's` text ?

Reply:
We override the 'text' property descriptors only for anchor and script element.